beetsplug/info.py

```diff
@@ -52,7 +53,11 @@ def emitter():
             tags[field] = getattr(mf, field)
         tags['art'] = mf.art is not None
         tags['path'] = displayable_path(path)
+        # create a temporary Item to take advantage of __format__
+        tags['item'] = Item(db=None, **tags)
```

Comment:
No problem, as a matter of fact thank you for making discussing these tiny-ish details and contributing to beets in general so enjoyable! I have just pushed a commit that makes the `emitters` return a `(dict, Item)` pair, adjusting `print_data()` (so it accepts an extra parameter, the `Item`) and the main loop accordingly.

I also took the liberty of revising how the `path` was handled: it is only used for printing it at the top of `print_data()` when not using a format string, just before the fields. Since we now have `Item.path` available at `print_data`, I thought it would make sense to remove some code that was mainly used for juggling with the `path` dict key (appending it to the dict on the emitters, preserving it from being deleted during `key_filter`, and popping it during `print_data`), hopefully resulting in a slightly cleaner code.